readme.md

```diff
@@ -76,13 +76,13 @@ Devices and data sources in an IoT solution can range from a simple network-conn
-| TPM Individual Enrollment   |  :heavy_minus_sign: |  :heavy_minus_sign: | :heavy_check_mark: | :heavy_multiplication_x: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Trusted Platform Module](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#trusted-platform-module-tpm).   Please visit the [samples folder](./provisioning/provisioning-samples/) and this [quickstart](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-tpm-java) on how to create a device client.  Websocket connection over AMQP is currently not available. TPM over MQTT is currently not supported by the Device Provisioning Service.                                                                                                                                                                                                              |
-| X.509 Individual Enrollment | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [X.509 leaf certificate](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#leaf-certificate).   Please visit the [samples folder](./provisioning/provisioning-samples/) and this [quickstart](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-x509-java) on how to create a device client. |
-| X.509 Enrollment Group      | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [enrollment group](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [X.509 root certificate](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#root-certificate).   Please visit the [samples folder](./provisioning/provisioning-samples/) to learn more about this feature.                                                                                                                                                                                            |
```

Comment:
This looks like a mess to read, but all I did was remove the '/' character at the end of links that were supposed to point to folders